pom.xml

```diff
@@ -264,6 +270,14 @@
 					<artifactId>findbugs-maven-plugin</artifactId>
 					<version>3.0.4</version>
 				</plugin>
+				<plugin>
+				    <groupId>org.apache.maven.plugins</groupId>
+				    <artifactId>maven-surefire-plugin</artifactId>
+				    <version>2.19.1</version>
+				    <configuration>
+				        <groups>tests.IntegrationTest</groups>
```

Comment:
with this configuration, only the integration tests are running. We want a configuration which will run unit tests and then the integration tests. A simple way seems to be
```shell
> mvn '-Dtest=unit.in.ac.bits.protocolanalyzer/**/*Test.java' test
> mvn '-Dtest=integration.in.ac.bits.protocolanalyzer/**/*Test.java' test
```
we can avoid the surefire plug-in altogether.

Reply:
The above mentioned commands can be put in travis.yml. We can replace [line-11: mvn test -B](https://github.com/prasadtalasila/BITS-Darshini/blob/separate-tests/.travis.yml#L11) with the above mentioned commands.